tests/features/paragraphs/latest_blog_posts_paragraphs.feature

```diff
@@ -4,59 +4,91 @@ Feature: Latest Blogs Paragraphs
 
   Background: Log in
     Given I am logged in as a user with the "Editor" role
+    And I create "taxonomy_term" of type "blog_category":
```

Comment:
Large background for all paragraphs is making tests dependent.  Better to split creation in each Case.